### PR TITLE
Implement rayyan-store support for new mobile features

### PR DESCRIPTION
**Pull request by fabubaker**
@hammady since I can't assign reviewers.

### CODE REVIEW

**Comment by fabubaker** [comment]
Forced-pushed to rebase on top of updated master.

File: server/src/adapters/postgres.js [hunk 1]
```diff
@@ -988,6 +1034,33 @@ module.exports = {
...
+      default:
+        throw new Error(`Unknown lastCustomization key: ${key}!`)
+    }
+
+    return pool.query(`SELECT created_at
+      FROM r${reviewId}_customizations_log
+      WHERE user_id=$1 AND ${condition}
+      ORDER BY created_at DESC LIMIT 1`, [userId]
+    ).then(function (res) {
+      if (res.rowCount === 1) return res.rows[0]
```
Comment (hammady) [inline server/src/adapters/postgres.js]: Return a string and wrap it as json in app.js.

File: server/src/adapters/postgres.js [hunk 2]
```diff
@@ -988,6 +1034,33 @@ module.exports = {
...
+      else return { created_at: '' }
```
Comment (fabubaker) [inline server/src/adapters/postgres.js]: Change `created_at` key name here.

File: server/src/adapters/postgres.js [hunk 3]
```diff
@@ -943,6 +943,52 @@ module.exports = {
...
+
+        if (type === 'at_least') {
+          inputArray.reverse()
+          id = userCounts
+          idAsc = false
+        }
+
+        var sum = 0
+        var outputArray = _.map(inputArray, function (count) {
+          sum += parseInt(count || 0)
```
Comment (hammady) [inline server/src/adapters/postgres.js]: Check if `parseInt` is necessary and remove if not.

File: server/src/adapters/postgres.js (hunk 2 above)
Comment (fabubaker) [reply server/src/adapters/postgres.js]: See 321d962.

File: server/src/adapters/postgres.js (hunk 1 above)
Comment (fabubaker) [reply server/src/adapters/postgres.js]: See 321d962.

File: server/src/adapters/postgres.js (hunk 3 above)
Comment (fabubaker) [reply server/src/adapters/postgres.js]: Wasn't necessary, removed in 28d4375.

**Pull request merged**